tests/common/orchestrator.rs

```diff
@@ -55,6 +55,8 @@ pub const ORCHESTRATOR_STREAM_CONTENT_DETECTION_ENDPOINT: &str =
 pub const ORCHESTRATOR_DETECTION_ON_GENERATION_ENDPOINT: &str = "/api/v2/text/detection/generated";
 pub const ORCHESTRATOR_CONTEXT_DOCS_DETECTION_ENDPOINT: &str = "/api/v2/text/detection/context";
 pub const ORCHESTRATOR_CHAT_DETECTION_ENDPOINT: &str = "/api/v2/text/detection/chat";
+pub const ORCHESTRATOR_CHAT_COMPLETIONS_DETECTION_ENDPOINT: &str =
```

Comment:
nit: Line break, so it lines up with the endpoints grouping in the [public API](https://foundation-model-stack.github.io/fms-guardrails-orchestrator/?urls.primaryName=Orchestrator+API#/).
```suggestion

pub const ORCHESTRATOR_CHAT_COMPLETIONS_DETECTION_ENDPOINT: &str =
```

Reply:
Applying suggestion